web/src/beta/components/StorytellingPageSectionItem/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC } from "react";
+
+import { styled, useTheme } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+type Props = {
+  icon: string;
+  title: string;
+  index: number;
+  active?: boolean;
+  onAction?: () => void;
+  onClick?: () => void;
+};
+
+const StorytellingPageSectionItem: FC<Props> = ({
+  icon,
+  title,
+  index,
+  active,
+  onAction,
+  onClick,
+}) => {
+  const theme = useTheme();
+  return (
+    <HorizontalBox>
+      <VerticalBox>
+        <Text size={"body"}>{index}</Text>
+        <Icon icon={icon} />
+      </VerticalBox>
+      <TitleArea active={active}>
+        <Text
+          onClick={onClick}
+          size={"footnote"}
```

Comment:
size="footnote" is enough. Don't need to wrap in {}

Reply:
Fix completed.